src/firebolt/service/V2/engine.py

```diff
@@ -115,6 +115,18 @@ def get_many(
             ]
             return [Engine._from_dict(_dict, self) for _dict in dicts]
 
+    @staticmethod
+    def _format_engine_parameter(
+        value: Union[str, int, EngineType, InstanceType, WarmupMethod]
+    ) -> Union[str, int]:
+        if isinstance(value, InstanceType):
+            return value.name
+        if isinstance(value, (EngineType, WarmupMethod)):
+            return str(value.value)
+        elif isinstance(value, (int, float)):
+            return int(value)
```

Comment:
I'm not sure about the float here. If someone passes `1.1` in scale, that's a misconfiguration and should explicitly fail. I think the only way this is ok if a float value has no decimal part e.g. `1.0`